src/gui/src/components/explorer-grid/selected-item/tabs-overview.tsx

```diff
@@ -15,14 +24,80 @@ export const OverviewTabButton = () => {
 }
 
 export const OverviewTabContent = () => {
-  const { selectedItemDetails, selectedItem } = useSelectedItem()
+  const { manifest } = useSelectedItem()
 
   return (
-    <TabsContent value="overview">
-      {selectedItem.to?.manifest?.description ?
-        <div className="px-6 py-4">
+    <TabsContent value="overview" className="flex flex-col gap-4">
+      <div
+        className={cn(
+          'flex flex-wrap gap-4 px-6 pt-4',
+          !manifest?.homepage && !manifest?.funding && 'hidden',
+        )}>
+        {manifest?.homepage && (
+          <div className="flex flex-col gap-1">
+            <p className="inline-flex items-center gap-1.5 text-sm text-muted-foreground">
+              <Globe size={16} className="text-blue-500" />
+              <span>Homepage</span>
+            </p>
+            <Link href={manifest.homepage} className="text-sm">
+              {manifest.homepage}
+            </Link>
+          </div>
+        )}
+        {manifest?.funding && (
+          <div className="flex flex-col gap-1">
+            <p className="inline-flex items-center gap-1.5 text-sm text-muted-foreground">
+              <HeartHandshake size={16} className="text-rose-400" />
+              <span>Funding</span>
+            </p>
+            {Array.isArray(manifest.funding) ?
+              manifest.funding.map((entry, index) => (
+                <span key={index}>
+                  {typeof entry === 'string' ? entry : entry.url}
+                </span>
```

Comment:
I think this one should also be a `<Link href=...` similar to the conditions below, I stumbled upon a package that uses it and it renders the wrong font and it can't be clicked: 

![Screenshot 2025-04-15 at 4 05 51 PM](https://github.com/user-attachments/assets/ecdbc14f-b80b-47f6-8cca-d9555872a64e)